WordPress/Classes/ViewRelated/System/WPTabBarController+ShowTab.swift

```diff
@@ -15,17 +15,21 @@ extension WPTabBarController {
             }
             return
         }
-
         guard let blog = inBlog ?? self.currentOrLastBlog() else { return }
+        guard content == nil else {
+            showPageEditor(blog: blog, title: title, content: content)
```

Comment:
From what I can tell the content property of this call is only ever populated from a deeplink that is no longer hooked up. (path = `newPage`) But I kept the support in place.

Reply:
If `content` is `nil` (e.g. via the deeplink), this calls the same function again and appears to lead to an infinite loop.

There was [a recent PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/13707) that implemented a deeplink to open a new Page in the editor. I tested these below links in the current WPiOS 15.5 version and they work, and then went to check if they still work using the Alpha build you provided on this PR.

```
https://apps.wordpress.com/get#/page
https:/apps.wordpress.com/get#/page/<domain>
```

The trouble is that the alpha build uses a different URL scheme so these links actually took me to the App Store to download WPiOS 15.5. I thought that these links might redirect to wpalpha://newpage?content=test (as seen on that PR), but these opened the Classic editor — so I hit a bit of a dead end.

I hope this has been helpful, sorry I couldn't come up with something conclusive here. Would you like to test more yourself?

Reply:
I updated the above to fix a typo (bolded) 😅

Reply:
I did miss that that PR it looks like we should be able to deploy locally and test using `wpdebug://newpage?content=test` I'll give this a try and see what happens. Thanks so much for sharing this 🙇 

That method signature is actually the same that I use for the path but swift figures it out. Looking at this though it is very confusing and unclear on how it will behave 😅 . I'll rename that function. I must have been too focused when working on this one.

Let me take another pass at this and I'll drop a comment when it's ready for revie again. Thanks so much @guarani :)